build_variables.bzl

```diff
@@ -822,6 +823,7 @@ libtorch_python_core_sources = [
     "torch/csrc/dynamo/init.cpp",
     "torch/csrc/functorch/init.cpp",
     "torch/csrc/jit/backends/backend_init.cpp",
+    "torch/csrc/jit/ir/graph_utils.cpp",
```

Comment:
I think you should remove this to fix the bazel build issue

Reply:
updated in the latest version